content-src/components/LinkMenuButton/LinkMenuButton.scss

```diff
@@ -6,7 +6,7 @@
   width: $link-menu-button-size;
   height: $link-menu-button-size;
   background-color: $white;
-  background-image: url('img/glyph-more-16.svg');
+  background-image: url($image-path + 'glyph-more-16.svg');
```

Comment:
I propose we leave the inlining for another PR, as I'd like to avoid this one growing any more.